packages/react-pose/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 React Pose adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.0] 2018-11-01
+
+### Removed
+
+- Removed support for `hostRef` and `innerRef` properties.
+
```

Comment:
Was 3.3.8 released? Would be good to release bug fixes without making this breaking change, 4.0 might follow immediately but that would give more time to people for migration

Reply:
Yeah it's been released